docs/resources/Guild.md

```diff
@@ -133,6 +133,8 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | COMMUNITY                        | guild can enable welcome screen, Membership Screening, stage channels and discovery, and receives community updates |
 | DISCOVERABLE                     | guild is able to be discovered in the directory                                                                     |
 | FEATURABLE                       | guild is able to be featured in the directory                                                                       |
+| HAS_DIRECTORY_ENTRY              | guild is in a [Student Hub](https://support.discord.com/hc/en-us/articles/4406046651927-Discord-Student-Hubs-FAQ)   |
+| HUB                              | guild is a [Student Hub](https://support.discord.com/hc/en-us/articles/4406046651927-Discord-Student-Hubs-FAQ)      |
```

Comment:
Should there be a second link? IMO having two links to the same thing right after each other is redundant.